models/stg_google_ads__ad_history.sql

```diff
@@ -52,11 +54,17 @@ url_fields as (
         {{ dbt.split_part('final_url', "'?'", 1) }} as base_url,
         {{ dbt_utils.get_url_host('final_url') }} as url_host,
         '/' || {{ dbt_utils.get_url_path('final_url') }} as url_path,
-        {{ dbt_utils.get_url_parameter('final_url', 'utm_source') }} as utm_source,
-        {{ dbt_utils.get_url_parameter('final_url', 'utm_medium') }} as utm_medium,
-        {{ dbt_utils.get_url_parameter('final_url', 'utm_campaign') }} as utm_campaign,
-        {{ dbt_utils.get_url_parameter('final_url', 'utm_content') }} as utm_content,
-        {{ dbt_utils.get_url_parameter('final_url', 'utm_term') }} as utm_term
+        --Extracting UTMs for url report's use 
+        coalesce( {{ dbt_utils.get_url_parameter('final_url', 'utm_source') }} ,
+                  {{ dbt_utils.get_url_parameter('final_url_suffix', 'utm_source') }} ) as utm_source,
```

Comment:
Would there ever be a scenario where both `final_urls` and `final_url_suffix` are populated in the source, but you want to leverage the value from `final_url_suffix`?

Reply:
In this case, I would think `final_url_suffix` should always take priority if defined. [As per Google documentation](https://support.google.com/searchads/answer/9072131?hl=en):

> To clearly distinguish between URL parameters that are used to determine the content of a page from parameters that are used for tracking purposes on your own site, and from parameters that are used by third-party trackers, Google Ads has introduced a new field named Final URL Suffix.

That description makes it clear to me that `final_url_suffix` is the de facto method for defining tracking parameters such as UTM.